### ISSUE TITLE
getRecursionsInPolygon() returns error with Move and MoveStack objects

### ISSUE THREAD

**Issue by sealavi** [cbracis/recurse#1]
getRecursionsInPolygon() returns the following error when trying to use with Move or MoveStack objects:

Error in as.difftime(revisitStats$timeSinceLastVisit) : 
  'tim' is not character or numeric

**Comment by yunusserhatbicakci** [comment]
what do you want to do? 
for example; 

I just want to show revisit stats in histogram:

`hist(as.numeric(samplePoligon$revisitStats$timeSinceLastVisit))`

**Comment by yunusserhatbicakci** [comment]
> getRecursionsInPolygon() returns the following error when trying to use with Move or MoveStack objects:
> 
> Error in as.difftime(revisitStats$timeSinceLastVisit) :
> 'tim' is not character or numeric

I think your problem is you don't have any point in the polygon. Are you sure your point data overlap your polygon?

**Comment by eldritchmayo** [comment]
Hello, I am having the same issue, and I am wondering if it has been resolved or just closed?

> > getRecursionsInPolygon() returns the following error when trying to use with Move or MoveStack objects:
> > Error in as.difftime(revisitStats$timeSinceLastVisit) :
> > 'tim' is not character or numeric
> 
> I think your problem is you don't have any point in the polygon. Are you sure your point data overlap your polygon?

I am certain that my points overlap with the polygone as it is a core area polygone derived via dBBMM in the `move` package from the same data.

My data looks like this: 
` `                    x           y              timestamp1       ID
12058 44.74746 20.28958 2021-06-20 01:02:08 Neda
12059 44.74687 20.28921 2021-06-20 04:00:37 Neda
12060 44.74498 20.28861 2021-06-20 07:00:37 Neda
12061 44.74487 20.28841 2021-06-20 10:00:37 Neda
12062 44.74500 20.28863 2021-06-20 13:00:38 Neda
12063 44.74482 20.28848 2021-06-20 16:00:37 Neda`

My polygones projection:
```
Coordinate Reference System:
Deprecated Proj.4 representation:
 +proj=aeqd +lat_0=44.7400932 +lon_0=20.29059525 +x_0=0 +y_0=0
+datum=WGS84 +units=m +no_defs
```

**Comment by cbracis** [comment]
I was never able to reproduce the error, but I would be happy to take another look if you can send me a reproducible example.

**Comment by eldritchmayo** [comment]
> I was never able to reproduce the error, but I would be happy to take another look if you can send me a reproducible example.


Certainly,

I would gladly sent you the original files and code. How can I do it directly? I can't seem to find the option on Github (I am fairly new here).

**Comment by cbracis** [comment]
no worries, you can send it to me at cbracis@uw.edu, the contact info in with the R package if you do ?recurse

**Comment by cbracis** [comment]
Fixed an issue with NA values sometimes being converted to difftime values which could cause this error. Fixed in version 1.4.
